chain/types/execresult.go

```diff
@@ -38,10 +40,11 @@ type ReturnTrace struct {
 }
 
 type ExecutionTrace struct {
-	Msg        MessageTrace
-	MsgRct     ReturnTrace
-	GasCharges []*GasTrace      `cborgen:"maxlen=1000000000"`
-	Subcalls   []ExecutionTrace `cborgen:"maxlen=1000000000"`
+	Msg          MessageTrace
+	MsgRct       ReturnTrace
+	InvokedActor *ActorTrace      `json:",omitempty"`
```

Comment:
Is this nullable to make it backward compatible, do we guarantee it exists on traces from this point forward? There's usage in `node/impl/full/eth_trace.go` that assumes it exists and will panic otherwise (3 instances of `et.InvokedActor.State.Code`).

Reply:
No, it can definitely not be set. Nice catch!

Reply:
Fixed by simply skipping these traces. It can only happen if we fail to resolve the target actor, and that's not a case that's even possible in the EVM (i.e., the EVM would have aborted the call before it even hit the tracing logic).